lib/ain-ocean/src/indexer/transaction.rs

```diff
@@ -41,8 +41,14 @@ pub fn index_transaction(ctx: &BlockContext, tx: Transaction, idx: usize) -> Res
     SERVICES.transaction.by_id.put(&tx_id, &trx)?;
     // Indexing transaction vin
     for (vin_idx, vin) in tx.input.iter().enumerate() {
+        let vout_bytes = vin.previous_output.vout.to_be_bytes();
         let trx_vin = TransactionVin {
-            id: format!("{}-{}", tx_id, vin_idx),
+            id: format!(
```

Comment:
Shouldn't be String here, we can leverage Column trait to pass a serializable tuple instead, i.e. `(txid, vin.previous_output.txid, vin.previous_output.vout.n)`